### PR TITLE
feat: bundle storage price change proposal

### PR DESCRIPTION
**Pull request by dckc**
goal:
closes :#10

 - [x] fetch stuff should use useEffect
 - [x] hardcoded to local chain. should use selected network

### CODE REVIEW

File: src/components/ParameterChangeForm.tsx [hunk 1]
```diff
@@ -1,21 +1,231 @@
...
+/**
+ * TODO: import from "@agoric/cosmic-proto/swingset/swingset.js"
+ * which uses camelCase
+ */
+type SwingsetParams = {
+  beans_per_unit: StringBeans[];
+  fee_unit_price: Coins[];
+  bootstrap_vat_config: string;
+  power_flag_fees: PowerFlagFee[];
+};
```
Comment (0xpatrickdev) [inline src/components/ParameterChangeForm.tsx:25]: these might need to be camel case depending how it's encoding. (We send in camel and they come out snake)

**Comment by dckc** [comment]
screenshots:

image

image

**Comment by dckc** [comment]
@0xpatrickdev PTAL. What do you think?

**Comment by dckc** [comment]
I animated the minimum deposit and current balance.

 - [ ] TODO: warn if deposit > balance

image

**Review by 0xpatrickdev — COMMENTED** [review]
Looks great i am going to add a commit that merges this with ProposalForm and cleans up the `msgType` type stuff. Are you at a stopping point?

**Review by 0xpatrickdev — APPROVED** [review]

**Pull request merged**